synapse/util/caches/expiringcache.py

```diff
@@ -95,6 +98,13 @@ def __getitem__(self, key):
 
         return entry.value
 
+    def pop(self, key, default=None):
+        value = self._cache.pop(key, SENTINEL)
```

Comment:
this is the same as `return self._cache.pop(key, default)` now.

docstring wouldn't go amiss.

Reply:
> this is the same as `return self._cache.pop(key, default)` now.

It's not, actually, as `self._cache.pop` won't raise if `default` is `None`. Thinking about it we can probably just do `return self._cache.pop(key, **kwargs)`

Reply:
Argh, I've completely messed this up